docs/settings/search-sessions-settings.asciidoc

```diff
@@ -0,0 +1,31 @@
+
+[[search-session-settings-kb]]
+=== Search sessions settings in {kib}
+++++
+<titleabbrev>Search sessions settings</titleabbrev>
+++++
+
+You do not need to configure any settings to use {kib} search session features. They are
+enabled by default.
+
+
+[cols="2*<"]
+|===
+| `search:sessions:enabled`
+  | Set to `true` (default) to enable search sessions.
+
+| `search:Sessions:trackingInterval`
+  | The frequency for updating the state of a search session. The default is 10s.
+
+| `search:Sessions:pageSize`
+  | The number of search sessions to show per page in *Stack Management*. The default is 10000.
+
+| `search:Sessions:defaultExpiration`
+  | How long search session results are stored before they are deleted.
+  Extending a search session resets the expiration by the same value. The default is 7d.
+
+| `search:sessions:maxUpdateRetries`
```

Comment:
I don't see why this option would be especially relevant here as well.
(There are many other options we don't list anyway, see `x-pack/plugins/data_enhanced/config.ts` for reference)